package.json

```diff
@@ -4,15 +4,15 @@
   "description": "Tool to debug failed/dropped requests at client side",
   "main": "src/requestsDebugger.js",
   "scripts": {
-    "start": "NODE_ENV=prod node src/requestsDebugger.js",
+    "start": "cross-env NODE_ENV=prod node src/requestsDebugger.js",
```

Comment:
do we need `cross-env` for `start` script?

I think we can have it only for `test` script since we have conditions in the code which checks for the `test` env equality. We don't check for `prod` anywhere...

Reply:
Passing NODE_ENV is important as it distinguishes environment in which code is running. Its very likely to be used in future if not in present. Its going to affect how code behaves in such scenario. Just added it so that npm run start works for user to run server in windows. Only commands working in windows and mac/linux after making above changes are install and start.

Reply:
will `npm test` work on windows now? We will need `cross-env` there as well right? I think `;` will also break there...

Reply:
Even after adding cross-env npm test command fails because nyc.js command is failing with 'invalid character' error.

Reply:
possible to fix?

Reply:
We can at least have `start, test and install` command working in windows...